.travis.yml

```diff
@@ -0,0 +1,25 @@
+---
+sudo: required
+dist: trusty
```

Comment:
Unless there's some magical way to test multiple distros so we could verify that it installs the package when available and clones the git when not...